src/config/services/managing.md

```diff
@@ -70,6 +70,18 @@ currently running:
 # rm /etc/runit/runsvdir/default/<service>
 ```
 
+### Testing services
+
+To check a service works as expected before fully enabling it:
+
+```
+# touch /etc/sv/<service>/down
+# ln -s /etc/sv/<service> /var/service
+# sv once <service>
+```
+
+If everything works, remove the `down` file to enable the service.
```

Comment:
I'd like to have some advice that they should remove the service either way, otherwise they will forget about it when trying to enable the service some time from now.

Reply:
Sorry, i don't follow? The use-case i'm thinking of is: "I want to enable this service, but I need to check it works first. Yes, it does. Okay, I can remove `down`, which will leave the service enabled."

Reply:
Your use case is the "happy path", so to speak. If the service has any issues, and the user decides to simply drop it, they should "remember to remove the `down` file as well, otherwise enabling the service at a later time won't actually start it"

Reply:
Ah, okay. How about something like: "If everything works, remove the \`down\` file to enable the service. If not, and you don't wish to enable the service, remove both the \`down\` file and the symlink in \`/var/service/\`."?

Reply:
Hmm, this is on the same page that has all the other information about service management. I re-read it and perhaps this warning isn't really necessary.

Reply:
So leave it as-is? If so, i'll force-push the other change, then i guess it's ready for you to merge. :-)

Reply:
I think so, yeah!